synapse/lib/stormlib/stix.py

```diff
@@ -607,6 +664,68 @@ def timestamp(self, tick):
         dt = datetime.datetime.utcfromtimestamp(tick / 1000.0)
         return dt.isoformat(timespec='milliseconds') + 'Z'
 
+@s_stormtypes.registry.registerLib
+class LibStixImport(s_stormtypes.Lib):
+    '''
+    A Storm Library for importing STIX version 2.1 data.
+    '''
+    _storm_locals = (  # type: ignore
+        {
+            'name': 'validate', 'desc': '''
+            Validate a STIX Bundle
+            ''',
+            'type': {
+                'type': 'function', '_funcname': 'validateBundle',
+                'args': (
+                    {'type': 'dict', 'name': 'bundle', 'desc': 'The stix bundle to validate.'},
+                ),
+                'returns': {'type': 'dict', 'desc': 'Results dictionary.'}
+            }
+        },
+        {
+        'name': 'lift', 'desc': '''
+        Lift nodes from a STIX Bundle made by Synapse.
+        ''',
+            'type': {
+                'type': 'function', '_funcname': 'liftBundle',
+                'args': (
+                    {'type': 'dict', 'name': 'bundle', 'desc': 'The stix bundle to lift nodes from.'},
+                ),
+                'returns': {'name': 'Yields', 'type': 'storm:node', 'desc': 'Yields nodes'}
+            }
+        },
+    )
+    _storm_lib_path = ('stix', 'import')
+
+    def __init__(self, runt, name=()):
+        s_stormtypes.Lib.__init__(self, runt, name)
+
+    def getObjLocals(self):
+        return {
+            'lift': self.liftBundle,
+            'validate': self.validateBundle,
+        }
+
+    async def validateBundle(self, bundle):
```

Comment:
This API should live on `$lib.stix` since it's not specific to import or export.